src/olympia/landfill/management/commands/generate_addons.py

```diff
@@ -40,6 +41,9 @@ def handle(self, *args, **kwargs):
         if not settings.DEBUG:
             raise CommandError('You can only run this command with your '
                                'DEBUG setting set to True.')
+
+        translation.activate(settings.LANGUAGE_CODE)
```

Comment:
This should be done using the [override](https://docs.djangoproject.com/en/1.8/ref/utils/#django.utils.translation.override) context manager instead to avoid changing anything in the caller if there is one.